### PR TITLE
Add cards for all Geotrend models

### PR DESCRIPTION
**Pull request by amineabdaoui**
# What does this PR do?

Adds model card

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: model_cards/Geotrend/bert-base-ar-cased/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+tags:
+- bert
+- ar
+- masked-lm
+- pytorch
+- tf
```
Comment (julien-c) [inline model_cards/Geotrend/bert-base-ar-cased/README.md:7]:
```suggestion
language: ar
```

This should be a language tag, and the other tags are automatic

**Review by julien-c — CHANGES_REQUESTED** [review]
Thanks for sharing! Any way you could update the tags like in my comment? Let me know if any help is needed

File: model_cards/Geotrend/bert-base-ar-cased/README.md (hunk 1 above)
Comment (amineabdaoui) [reply model_cards/Geotrend/bert-base-ar-cased/README.md]: Corrected in https://github.com/huggingface/transformers/pull/8617/commits/cf75587e11f76b5a3d1dcd90fb7c612704dd3d70
Thanks for the review @julien-c

**Comment by julien-c** [comment]
Awesome, thank you!

**Pull request merged**